include/lsst/meas/base/PixelFlags.h

```diff
@@ -42,13 +41,20 @@ namespace lsst { namespace meas { namespace base {
  */
 class PixelFlagsControl {
 public:
-
+    LSST_CONTROL_FIELD(center, std::vector<std::string>,
+                       "List of mask planes for which to search center of footprint");
+    LSST_CONTROL_FIELD(any, std::vector<std::string>,
+                       "List of mask planes for which to search entire footprint");
     /**
      *  @brief Default constructor
      *
      *  All control classes should define a default constructor that sets all fields to their default values.
      */
-    PixelFlagsControl() {}
+    // We define this string to be consistent with the port from HSC. It also
+    // provideds a single place to change the string if a change is needed.
+    const std::string fname;
```

Comment:
I don't think this is necessary. Elsewhere, we hard-code flag fields to be called `flag`, and I'm not sure we gain anything by not doing so here.